app/components/geolocation.rb

```diff
@@ -0,0 +1,6 @@
+module Geolocation
+  def find_by_ip_address(ip_address)
```

Comment:
this is an entry point to the `Geolocation` component for the outside world. For example, this method is being used in the API controller to hide defails of using repositories, etc.